website/docs/modules/gather_batch_evidence.md

```diff
@@ -5,25 +5,205 @@ sidebar_position: 4
+  The content of the files may read as the following, 
+  and the format is explained [on this page](https://www.htslib.org/doc/faidx.html).
+
+  ```bash
+  chrX    156040895       2903754205      100     101
+  chrY    57227415        3061355656      100     101
+  ```
+
+  You may use the following files for these fields:
+
+  ```json
+  "cnmops_chrom_file": "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/autosome.fai"
+  "cnmops_allo_file": "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/allosome.fai"
+  ```
+  
+- `cnmops_exclude_list`: You may use the following file for this field.
+  ```
+  gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/GRCh38_Nmask.bed
+  ```
+
+#### GATK-gCNV inputs
+
+The following inputs are configured based on the outputs generated in the [`TrainGCNV`](./gcnv) workflow.
+
+- `contig_ploidy_model_tar`: [`TrainGCNV.cohort_contig_ploidy_model_tar`](./gcnv#contig-ploidy-model-tarball)
+- `gcnv_model_tars`: [`TrainGCNV.cohort_gcnv_model_tars`](./gcnv#model-tarballs)
+
+
+The workflow also enables setting a few optional arguments of gCNV.
+The arguments and their default values are as the following,
+and each argument is documented on 
+[this page](https://gatk.broadinstitute.org/hc/en-us/articles/360037593411-PostprocessGermlineCNVCalls)
+and
+[this page](https://gatk.broadinstitute.org/hc/en-us/articles/360047217671-GermlineCNVCaller).
+
+```json
+"gcnv_caller_internal_admixing_rate": 0.5,
+"gcnv_caller_update_convergence_threshold": 0.000001,
+"gcnv_cnv_coherence_length": 1000,
+"gcnv_convergence_snr_averaging_window": 100,
+"gcnv_convergence_snr_countdown_window": 10,
+"gcnv_convergence_snr_trigger_threshold": 0.2,
+"gcnv_copy_number_posterior_expectation_mode": "EXACT",
+"gcnv_depth_correction_tau": 10000,
+"gcnv_learning_rate": 0.03,
+"gcnv_log_emission_sampling_median_rel_error": 0.001,
+"gcnv_log_emission_sampling_rounds": 20,
+"gcnv_max_advi_iter_first_epoch": 1000,
+"gcnv_max_advi_iter_subsequent_epochs": 200,
+"gcnv_max_training_epochs": 5,
+"gcnv_min_training_epochs": 1,
+"gcnv_num_thermal_advi_iters": 250,
+"gcnv_p_alt": 0.000001,
+"gcnv_sample_psi_scale": 0.000001,
+"ref_copy_number_autosomal_contigs": 2
+```
+
+
+#### Docker images
+
+The workflow needs the following Docker images, which you may find a link to their 
+latest images from [this file](https://github.com/broadinstitute/gatk-sv/blob/main/inputs/values/dockers.json).
+
+  - `cnmops_docker`;
+  - `condense_counts_docker`;
+  - `linux_docker`;
+  - `sv_base_docker`;
+  - `sv_base_mini_docker`;
+  - `sv_pipeline_docker`;
+  - `sv_pipeline_qc_docker`;
+  - `gcnv_gatk_docker`;
+  - `gatk_docker`.
+
+#### Static inputs
+
+You may refer to [this reference file](https://github.com/broadinstitute/gatk-sv/blob/main/inputs/values/resources_hg38.json)
```

Comment:
I like this version better than including the file paths directly on the website. It might be more efficient to just have a general section describing where to find the reference resources rather than saying this for every input, though

Reply:
We'll improve the organization, it will take a few iterations.